bin/webpack-dev-server.js

```diff
@@ -97,6 +98,10 @@ const config = require('webpack-cli/bin/convert-argv')(yargs, argv, {
 // we should use portfinder.
 const DEFAULT_PORT = 8080;
 
+// Try to find unused port and listen on it for 3 times,
+// if port is not specified in options.
+const defaultPortRetry = defaultTo(tryParseInt(process.env.DEFAULT_PORT_RETRY), 3);
```

Comment:
Why don't use `parseInt`?

Reply:
Mainly because NaN == null is false, so if parseInt returns NaN, defaultTo will not work properly.

Reply:
@u9520107 can we add comment in code about this?